openlibrary/accounts/model.py

```diff
@@ -699,6 +699,7 @@ def audit_accounts(email, password, require_link=False,
                     ia_account.itemname, email, password,
                     displayname=ia_account.screenname,
                     verified=True, retries=5, test=test)
+                ol_account.save_preferences({'public_readlog':'yes'})
```

Comment:
Actually, close, but I think I'm not right there either :)

We apparently have to get the account object using accounts/__init__.py `find`; sometihing like this:

https://github.com/internetarchive/openlibrary/blob/6846e736a8994f11bd980d82598c53bbd0d92b3f/openlibrary/accounts/model.py#L707:

```
            account = accounts.find(email=email)
            account.save_preferences({'public_readlog':'yes'})
```

I can test this with you tomorrow or Tuesday.

Reply:
@mekarpeles Whichever time suits you :)

Reply:
```python
            account = accounts.find(email=email)
            if account:
                account.save_preferences({'public_readlog': 'yes'})
```

Reply:
This line has been removed altogether but thank you for your feedback :)